src/main/resources/application.properties

```diff
@@ -1,8 +1,7 @@
-spring.jpa.properties.hibernate.hbm2ddl.auto=create
+#spring.jpa.properties.hibernate.hbm2ddl.auto=create
```

Comment:
usunęłaś tą propertkę dlatego, że ja tak powiedziałem czy z jakiegoś konkretnego powodu?? :)

Reply:
zrobiłam z automatu, później sobie uświadomiłam, że to przecież migracje... :D